spec.md

```diff
@@ -281,6 +281,27 @@ help intermediate gateways determine how to route the events.
 As defined by the term [Data](#data), CloudEvents MAY include domain-specific
 information about the occurrence. When present, this information will be
 encapsulated within the `data` attribute.
+The `dataref` attribute can be used to reference another location where this
```

Comment:
I like the direction of this. My comments are editorial in nature:
- it feels like most of this should be put under the `dataref` section instead of here because most of it applies to only when dataref is present. Then we could just add a pointer to the dataref attribute from here.
- s/can/MAY/ in this sentence

Reply:
Thanks for the careful review!

I have moved everything except the paragraph that describes the interaction between `data` and `dataref` into the `dataref` section: https://github.com/cloudevents/spec/pull/377/commits/3b923d4a6b55a77c316f0a887e0d987749bd1e9c